### ISSUE TITLE
Prototype - Spacing - Specificity issue when using breakpoints

### ISSUE THREAD

**Issue by mathieuhays** [zurb/foundation-sites#11710]
## What should happen?

`medium-margin-top-0` should override `margin-top-2` on medium screens and larger.

## What happens instead?

`medium-margin-top-0` doesn't override `margin-top-2` on medium screens and larger due to specificity issues.

Classes are defined as follows:
```
.margin-top-0 {}
@media(/* medium breakpoint condition */) {
  .medium-margin-top-0{}
}

/* ... */

.margin-top-2 {}
@media(/* medium breakpoint condition */) {
  .medium-margin-top-2{}
}
```

## Possible Solution

Make sure all levels are defined for the given breakpoints before moving on to the next one.

to be as follows:
```
.margin-top-0{}
.margin-top-1{}
.margin-top-2{}

@media(/* medium breakpoint condition */) {
  .medium-margin-top-0{}
  .medium-margin-top-1{}  
  .medium-margin-top-2{}
}
```

## Test Case and/or Steps to Reproduce (for bugs)

Test Case: https://mathieuhays.github.io/foundation-spacing-specificity-issue/

How to reproduce:
1. Using a similar setup as the starter project use the following setting line: `$global-prototype-breakpoints: $breakpoints;`
2. add the following classes together `margin-top-2 medium-margin-top-0`
3. Observe on a medium screen or larger

## Context

Settings line corresponding to the prototype breakpoints is set to the following:
`$global-prototype-breakpoints: $breakpoints;`

## Your Environment

- Foundation version(s) used: 6.4.3
- Browser(s) name and version(s): Firefox 65.0.1
- Device, Operating System and version:  Mac Os 10.14.3
- Link to your project: 

## Checklist

- [x] I have read and follow the CONTRIBUTING.md document.
- [x] There are no other issues similar to this one.
- [x] The issue title and template are correctly filled.

**Comment by DanielRuf** [comment]
Hi, thanks for reporting this.

Normally `@media` creates a higher specificity.

Does the same still happen in 6.5.3?

Can you provide a codepen which reproduces this?

**Comment by mathieuhays** [comment]
I've updated the test-case to use 6.5. The initial test case was using the develop branch, I now use develop-v6.5. The develop branch seems to be v6.4.3 and develop-v6.5 seems to be 6.5.1.

Test case url:
https://mathieuhays.github.io/foundation-spacing-specificity-issue/

Test case repo:
https://github.com/mathieuhays/foundation-spacing-specificity-issue

Which branch should I pull for the latest changes?

I initially found this bug using 6.5.3 while working on a website.

I tried to create a codepen for this but the versions of foundation available on CDN are using default config without the required change to the `_settings.scss` file as indicated above.
